#CAMERA INTRINSIC MATRIX
cameraIntrinsic: 1035.203305928913, 0, 668.4512727002184; 0, 1033.481997796407, 478.0281378365561; 0, 0, 1

#CAMERA DISTORTION MATRIX
cameraDistortion: -0.4096880681395127, 0.2088728091384491, -0.0003398759271267612, -0.0008781290070450316, -0.05624067494923116

#CAMERA FOVs IN RADIANS 
cameraVerticalFOV_0: 1.57
cameraHorizontalFOV_0: 1.57
